prometheus-operator/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+NAMESPACE=monitoring
+
+# Create the namespace for prometheus-operator
+kubectl create namespace ${NAMESPACE}
+
+# Create CRDs due to a bug. More information on https://github.com/helm/charts/tree/master/stable/prometheus-operator#helm-fails-to-create-crds
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/alertmanager.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheus.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/podmonitor.crd.yaml
```

Comment:
These should at least use versions of these files from a commit/release. Otherwise we have an unstable application in the marketplace (something breaks on master, customers install it here and it doesn't work, etc).